tests/test_xml_asserts.py

```diff
@@ -193,3 +193,13 @@ def test_assertXmlCompatible_incompatible_less_tags(self):
             self.assertXmlCompatible(old, new)
         except self.failureException as e:
             self.assertIn('Children length differs', str(e))
+
+    def test_assertXmlEqual_with_similar_children(self):
+        xml_string = ('<a><b><c><d>1</d><e>2</e></c><f><g>3</g></f></b><h>4</h><i><l>5</l><m>6</m></i>'
+                      '<i><l>7</l><m>8</m></i></a>')
+        xml = etree.fromstring(xml_string)
+
+        try:
+            self.assertXmlEqual(xml_string, xml)
+        except self.failureException as e:
+            self.fail('XML should be equals (Reported error: "{}")'.format(e))
```

Comment:
`XML should be equals (Reported error: "XML documents are not equal: Tags do not match: ed and e (path: /a/b/c/ed)")`
vs
`Reported error: "XML documents are not equal: Tags do not match: ed and e (path: /a/b/c/ed)"`